governance/polls/Ratification Poll - Modify Dai Foundation Core Unit Budget (DAIF-001) - August 8, 2022.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Ratification Poll for Modify Dai Foundation Core Unit Budget (MIP40c3-SP74) - August 8, 2022 
+summary: This Budget Proposal for DAI expenses aims to fund the activities of the DAI Foundation Core Unit (DAIF-001) as described in MIP39c2-SP17 adds Core Unit DAIF-001: Dai Foundation.
```

Comment:
Summary seems wrong, this isn't adding DAIF-001 and also can't have a colon in the yaml

Reply:
![image](https://user-images.githubusercontent.com/72349134/183444721-ab0a0be6-4805-4e6c-a283-7ece85eef9cb.png)

That's the Sentence Summary. I'm modifying it a bit for more clarity